MIGRATION.md

```diff
@@ -88,6 +89,52 @@ addParameters({
 });
 ```
 
+## Options addon deprecated
+
+In 4.x we added story parameters. In 5.x we've deprecated the options addon in favor of global parameters, and we've also renamed some of the options in the process (though we're maintaining backwards compatibility until 6.0).
+
+Here's an old configuration:
+
+```js
+addDecorator(
+  withOptions({
+    name: 'Storybook',
+    url: 'https://storybook.js.org',
+    goFullScreen: false,
+    addonPanelInRight: true,
+  })
+);
+```
+
+And here's its new counterpart:
+
+```js
+import { create } from '@storybook/theming';
+addParameters({
+  options: {
+    theme: create({
+      brandTitle: 'Storybook',
+      brandUrl: 'https://storybook.js.org'
+    }, 'normal'),
+    isFullscreen: false,
+    panelPosition: 'right',
+  }
+})
+
+Here is the mapping from old options to new:
+
+| Old               | New              |
+| ----------------- | ---------------- |
+| name              | theme.brandTitle |
+| url               | theme.brandUrl   |
+| goFullScreen      | isFullscreen     |
+| showStoriesPanel  | showNav          |
+| showAddonPanel    | showPanel        |
+| showSearchBox     |                  |
+| addonPanelInRight | panelPosition    |
+| selectedPanel     |                  |
```

Comment:
Are the ones that are missing here no longer supported?

Reply:
Updated